ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -148,6 +148,7 @@ l2leaf:
   node_groups:
     DC1_L2LEAF1:
       mlag: false
+      esi: true
```

Comment:
Why is this needed? Could we not just detect the `short_esi`? Espcially if we move it to `node_group`. Thanks

Reply:
Removed as per commit e3c7310